standardtracer/propagation.go

```diff
@@ -159,53 +222,52 @@ func (t *tracerImpl) JoinTraceFromBinary(
 	), nil
 }
 
-func (t *tracerImpl) JoinTraceFromText(
+const (
+	tracerStateHeaderName = "Tracer-State"
+	traceAttrsHeaderName  = "Trace-Attributes"
+)
+
+func (p goHTTPPropagator) InjectSpan(
```

Comment:
this one is marginally interesting... just shows how an injector and extractor can defer to other injectors and extractors to good effect.